packages/mdc-dialog/foundation.ts

```diff
@@ -68,6 +68,8 @@ export class MDCDialogFoundation extends MDCFoundation<MDCDialogAdapter> {
   private scrimClickAction_ = strings.CLOSE_ACTION;
   private autoStackButtons_ = true;
   private areButtonsStacked_ = false;
+  // Element to focus on after dialog has finished opening.
+  private initialFocusEl_: HTMLElement|null = null;
```

Comment:
nit: `private initialFocusEl_!: HTMLElement|null`;

Reply:
Not needed since null is valid here after initialization.